src/components/Overlay.js

```diff
@@ -3,10 +3,25 @@ import styled from "styled-components";
 import Settings from "./Settings";
 import RetreatButton from "./RetreatButton";
 
+const Background = styled.div`
+  position: absolute;
+  z-index: -1;
+  width: 100vw;
```

Comment:
`width` is assigned two times, see line 12

Reply:
Fixed